Tools/BlinkLocomotionTool/Scripts/WorldScaleVisuals.cs

```diff
@@ -0,0 +1,81 @@
+﻿using UnityEngine;
+using UnityEngine.Experimental.EditorVR.Tools;
+using UnityEngine.Experimental.EditorVR.Utilities;
+using UnityEngine.UI;
+
+public class WorldScaleVisuals : MonoBehaviour, IUsesCameraRig
```

Comment:
Mark this internal + any other new classes you've created
Add a proper namespace for this to live in.